cpg-language-python/src/test/resources/python/loadIncludes/stdlib/__init__.pyi

```diff
@@ -0,0 +1,2 @@
+class str(Sequence[str]):
```

Comment:
The builtins for python are in the `builtins` package. https://docs.python.org/3/library/builtins.html

Reply:
I now moved the builtin file selection to a trait, and based it on all analyzed files. The resolution of the functions in the builtins does not work anymore as we decided to do that mapping in the SymbolSolver in another PR

Reply:
I unresolved this because I fear we need another solution / method of the interface. For the symbol resolver we need to know which namespace contains the built-in. Can we model it this way that the function returns a Name/String and we then resolve the bultins.py based on the name “bultins”.

Reply:
Yes, this should be doable, one issue that we still have is that the extension may be ".py" or ".pyi" or even something else in the future if we build our own library representing file format. I will add the file extensions in the translation manager.

Reply:
> Yes, this should be doable, one issue that we still have is that the extension may be ".py" or ".pyi" or even something else in the future if we build our own library representing file format. I will add the file extensions in the translation manager.

But don't we already have the logic to translate a "package name" into a "filename" (in the python frontend), can't we just invoke that for the built-in name?

Reply:
I could incorporate that function if you could point me to it.

Reply:
I moved the trait to provide the builtin namespace instead of a boolean function that decides if a file is a builtin function. I also extracted the logic to decide if a file is a fitting candidate for a namespace to another function and used that to identify the builtins file and files, relevant when analyzing an import.

Reply:
Sounds good, only small comment about function vs. variable.